<!DOCTYPE html>
<html lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Bioinformatics Training Materials &middot; The Maine INBRE Program
    
  </title>

  <!-- CSS -->
  <link rel="stylesheet" href="/public/css/poole.css">
  <link rel="stylesheet" href="/public/css/syntax.css">
  <link rel="stylesheet" href="/public/css/lanyon.css">
  <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700%7CPT+Sans:400">

  <!-- Icons -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-precomposed.png">
  <link rel="shortcut icon" href="/public/favicon.ico">

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/atom.xml">
</head>


  <body class="theme-base-MDIBL">

    <!-- Target for toggling the sidebar `.sidebar-checkbox` is for regular
     styles, `#sidebar-checkbox` for behavior. -->
<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">

<!-- Toggleable sidebar -->
<div class="sidebar" id="sidebar">
  <div class="sidebar-item">
    <p>Maine INBRE</p>
  </div>

  <nav class="sidebar-nav">
    <a class="sidebar-nav-item" href="/">Home</a>

    

    
    
      
        
      
    
      
        
          <a class="sidebar-nav-item" href="/about/">About</a>
        
      
    
      
    
      
    
      
        
          <a class="sidebar-nav-item active" href="/tutorials/">Bioinformatics Training Materials</a>
        
      
    

    <!--<a class="sidebar-nav-item" href="/archive/v1.0.0.zip">Download</a>
    <a class="sidebar-nav-item" href="">GitHub project</a>
    <span class="sidebar-nav-item">Currently v1.0.0</span>-->
  </nav>

  <div class="sidebar-item">
    <p>Funding provided by the <a href="https://www.nigms.nih.gov/research/crcb/IDeA/Pages/default.aspx">National Institute of General Medical Studies</a> and the <a href="https://www.nih.gov">National Institute of Health</a>. <br/> Hosted on <a href="https://github.com/GreenwoodLab">Github</a>. <br/> Powered by <a href="http://jekyllrb.com/">jekyll</a>, <a href="http://getpoole.com/">poole</a>, and 
          <a href="http://yihui.name/knitr/">knitr</a>. <br/> Maintained by <a href="https://kylescotshank.github.io">Kyle Scot Shank</a></p>
  </div>
</div>


    <!-- Wrap is the content to shift when toggling the sidebar. We wrap the
         content to avoid any CSS collisions with our real content. -->
      <div class="wrap">
        <div class="masthead">
          <div class="container">
            <h3 class="masthead-title">
              <a href="/" title="Home">The Maine INBRE Program</a>
            <small>Training Materials Repository</small>
          </h3>
        </div>
      </div>

      <div class="container content">
        <div class="page">
  <h1 class="page-title">Bioinformatics Training Materials</h1>
  <p>On this page you will find a variety of curated training materials and tutorials related to bioinformatics, biostatistics, and general computational biology. They are broken down into two groups:</p>

<ul>
  <li><a href="#general-tutorials">General Tutorials</a>
    <ul>
      <li>These are general overviews of a subject, code, or software.</li>
    </ul>
  </li>
  <li><a href="#project-specific-materials">Project-specific Materials</a>
    <ul>
      <li>These are more specific training materials created in support of a particular project or course.</li>
    </ul>
  </li>
</ul>

<h2 id="general-tutorials">General Tutorials</h2>

<ul>
  <li><a href="https://github.com/MaineINBRE/IntroToShell">Introduction to the Unix Shell</a>
    <ul>
      <li>This tutorial is a modified form of the introduction to the Unix shell as offered by <a href="https://software-carpentry.org">Software Carpentry</a>. It focuses on teaching undergraduates the basic skills necessary to understand the shell and utilize command line tools.</li>
      <li>Key Skills: Unix, Bash, Command Line</li>
    </ul>
  </li>
  <li><a href="https://github.com/MaineINBRE/IntroToFASTQCandTrimmomatic">Introduction to Quality Assessment, FastQC, and Trimmomatic</a>
    <ul>
      <li>This tutorial is an introduction to the quality assessment step in a bioinformatics pipeline. It focuses on the most common file formats (FASTA/FASTQ), the motivation for and use of FastQC, and using Trimmomatic to trim paired-end reads. It is highly tailored to the use of the MDIBL Bioinformatics Core cluster to use example data.</li>
      <li>Key Skills: FastQC, Trimmomatic</li>
    </ul>
  </li>
  <li><a href="https://github.com/MaineINBRE/IntroToTopHat2">Introduction to TopHat</a>
    <ul>
      <li>This tutorial is an introduction to the <a href="https://ccb.jhu.edu/software/tophat/index.shtml">TopHat2</a> software suite and its use in mapping RNA-seq reads. Though an incredible tool in any sequence analysis pipeline, this particular tutorial is highly tailored to the use of the MDIBL Bioinformatics Core.</li>
      <li>Key Skills: TopHat</li>
    </ul>
  </li>
  <li><a href="https://github.com/MaineINBRE/IntroToHTSEQ">Introduction to HTseq</a>
    <ul>
      <li>This tutorial is an introduction to the <code class="highlighter-rouge">htseq-count</code> function in the <a href="http://www-huber.embl.de/users/anders/HTSeq/doc/index.html">HTseq</a> computational environment. Though useful in many situations, it is highly tailored to the use of the MDIBL Bioinformatics Core.</li>
      <li>Key Skills: HTseq, Samtools</li>
    </ul>
  </li>
  <li><a href="https://github.com/MaineINBRE/IntroToNCBIandBLAST">Introduction to NCBI and BLAST</a>
    <ul>
      <li>This tutorial was created for undergraduate students studying with Dr. Elizabeth Ehrenfeld at the <a href="https://www.smccme.edu">Southern Maine Community College</a>. It focuses primarily on learning how to navigate the NCBI website, perform simple queries, and understanding and perfomring BLAST alignments on simple sequences.</li>
      <li>Key Skills: BLAST, Entrez Search</li>
    </ul>
  </li>
  <li><a href="https://github.com/MaineINBRE/GenBankToGTF">Converting a Genbank file to a GTF file</a>
    <ul>
      <li>This script was created to show how to quickly convert a raw Genbank file to a GTF file when a GTF file cannot be located. It was created for undergraduate students at <a href="www.umaine.edu">The University of Maine</a>.</li>
      <li>Key Skills: perl</li>
    </ul>
  </li>
  <li><a href="https://github.com/MaineINBRE/Trinity2.8.4Marconi">De Novo Transcriptome Assembly</a>
  </li>
</ul>
<hr />

<h2 id="project-specific-materials">Project-specific Materials</h2>

<ul>
  <li><a href="https://github.com/MaineINBRE/GilesRNASeqGuide">Guide to RNA-Seq Analysis with Galaxy: pt.1</a>
    <ul>
      <li>This is a set of instructional materials for aligning RNA-seq reads from the <em>Giles</em> bacteriophage to the genome of its host, <em>Mycobacterium smegmatis</em>. This was done in support of the <a href="https://honors.umaine.edu">The University of Maine’s Honors College</a>. The Github repo also contains a link to an RPub page.</li>
      <li>Key Skills: Galaxy</li>
    </ul>
  </li>
  <li><a href="https://github.com/MaineINBRE/GilesRNASeqGuide2">Guide to RNA-Seq Analysis with Galaxy: pt.2</a>
    <ul>
      <li>This is a continuation of the above project. This is a set of instructional materials for aligning RNA-seq reads from the <em>Giles</em> bacteriophage to its own annotated genome. This was done in support of the <a href="https://honors.umaine.edu">The University of Maine’s Honors College</a>. The Github repo also contains a link to an RPub page.</li>
      <li>Key Skills: Galaxy</li>
    </ul>
  </li>
  <li><a href="https://github.com/MaineINBRE/GilesDESEQ2Guide">Using DESeq2 to Perform Basic Differential Expression Analysis</a>
    <ul>
      <li>This is a set of instructional materials for performing basic differential analysis on RNA-seq reads. It is explicitly designed to be the final module for the work on the <em>Giles</em> bacterioohage above. The Github repo also contains a link to an RPub page.</li>
      <li>Key Skills: R, Bioconductor, DESeq2</li>
    </ul>
  </li>
  <li><a href="https://github.com/MaineINBRE/Trinity2.8.4Marconi">Lobster De Novo Transcriptome Assembly</a>
  </li>
</ul>
</div>

<script type="text/javascript"
    src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>

      </div>
    </div>

    <label for="sidebar-checkbox" class="sidebar-toggle"></label>

    <script>
      (function(document) {
        var toggle = document.querySelector('.sidebar-toggle');
        var sidebar = document.querySelector('#sidebar');
        var checkbox = document.querySelector('#sidebar-checkbox');

        document.addEventListener('click', function(e) {
          var target = e.target;

          if(!checkbox.checked ||
             sidebar.contains(target) ||
             (target === checkbox || target === toggle)) return;

          checkbox.checked = false;
        }, false);
      })(document);
    </script>
  </body>
</html>

